sql/core/src/test/resources/sql-tests/inputs/window.sql

```diff
@@ -250,4 +250,16 @@ WINDOW w AS (
   ORDER BY salary DESC
   RANGE BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING
 )
-ORDER BY department;
\ No newline at end of file
+ORDER BY department;
+
+SELECT
+    employee_name,
+    salary,
+    first_value(employee_name) OVER w highest_salary,
+    nth_value(employee_name, 2) OVER w second_highest_salary
+FROM
+    basic_pays
+WINDOW
+    w AS (ORDER BY salary DESC ROWS BETWEEN UNBOUNDED PRECEDING AND 1 FOLLOWING),
```

Comment:
is the window spec name case insensitive?

Reply:
I checked the PgSQL and find the alias of window frame is case insensitive.